sceptre/diffing/stack_differ.py

```diff
@@ -169,7 +169,10 @@ def _extract_parameters_from_generated_stack(self, stack: Stack) -> dict:
         for key, value in stack.parameters.items():
             if isinstance(value, list):
                 value = ",".join(item.rstrip("\n") for item in value)
-            formatted_parameters[key] = value.rstrip("\n")
+            if isinstance(value, str):
+                formatted_parameters[key] = value.rstrip("\n")
+            elif value is None:
+                formatted_parameters[key] = value
```

Comment:
So I know we talked about this, but I just want to triple-check here: If you have created a stack by passing `None` as a parameter, when we describe the stack later, those same parameter values come back as None? Because if they come back as empty strings, this will result in a diff when there is none.